### DISCUSSION TITLE
What're your thoughts on porting some (convenience) functions from Make?

### DISCUSSION THREAD

**Discussion by gyreas** [casey/just#1997] [Ideas]
Hi.
There are useful functions from Make that will make things easier in Just. F.e, `prefix`/`suffix` which adds a prefix/suffix to a space delimited string of whatnot, or `wildcard` which selects files that match a (glob)pattern. What do you think about porting some of them over?

**Comment by casey** [comment]
I'm pretty open to adding functions, as long as they make sense. Feel free to open an issue proposing whatever you think might be useful. It's probably easiest to keep each individual PR/issue scoped to one function, or a group of related function, to keep review easy.
